WordPress/src/main/java/org/wordpress/android/ui/stats/StatsActivity.java

```diff
@@ -545,8 +548,10 @@ public void onClick(DialogInterface dialog, int id) {
     @Override
     public boolean onCreateOptionsMenu(Menu menu) {
         super.onCreateOptionsMenu(menu);
-        MenuInflater inflater = getMenuInflater();
-        inflater.inflate(R.menu.stats, menu);
+        if (!mNoMenuDrawer) {
```

Comment:
I think this is not needed here, otherwise the "View full site" will not be available.

![screen shot 2015-01-21 at 18 54 57](https://cloud.githubusercontent.com/assets/518232/5841693/1f1df61e-a19f-11e4-920a-1f148654fbf7.png)
![screen shot 2015-01-21 at 18 55 55](https://cloud.githubusercontent.com/assets/518232/5841699/271a213a-a19f-11e4-8b66-c105b8cb379b.png)